src/app/example_local.ml

```diff
@@ -138,12 +145,23 @@ let example () =
             "https://storage.googleapis.com/hammerlab-biokepi-data/precomputed/b37_20161007.tgz"
             ~in_directory:(bw#mount // "workdir/reference-genome");
         ]) in
+  let extra_services =
+    match conf_opt "epidisco_dev" with
+    | None -> []
+    | Some path ->
+      [
+        Docker_compose.Configuration.service ~image:"hammerlab/keredofi:epidisco-dev"
+          ~volumes:[sprintf "%s:/epidisco-shared/" path]
+          ~start_up_script:"#!/bin/sh\nwhile true ; do sleep 42 ; done\n"
```

Comment:
hmm, I am assuming 42 is just arbitrary and the while loop is to keep it sleeping as long as it takes?

Docker people use a binary called [pause](https://github.com/kubernetes/kubernetes/blob/master/build/pause/pause.c) to achieve this. https://github.com/kubernetes/kubernetes/blob/master/build/pause/Dockerfile

AFAIK, this is a small binary that just waits for a pause signal and once it has it, it silenty dies (which is cure, if you ask me :)) Not saying we should do that, but maybe for the future we can start using it.